Python/wrfrun/cycling.py

```diff
@@ -38,12 +38,10 @@
 if 'INIDIR' in os.environ:
   IniDir = os.environ['INIDIR'] # where the step file is found
 else: IniDir = os.getcwd() + '/' # current directory
-if 'DATATYPE' in os.environ:
-  if os.environ['DATATYPE'][:4] == 'CESM': lly = False # CESMx does not have leap-years
-  elif os.environ['DATATYPE'][:4] == 'CCSM': lly = False # CCSMx does not have leap-years
-  elif os.environ['DATATYPE'][:5] == 'CMIP5': lly = False # CCSMx does not have leap-years
-  else: lly = True # reanalysis have leap-years
-else: lly = False # GCMs like CESM/CCSM generally don't have leap-years
+if os.environ['LLEAP'] == '':
+  lly = True
+elif os.environ['LLEAP'] == '--noleap':
```

Comment:
lol... I did not realize that this is the value it has in setupExperiment... I guess that's a command line argument... I guess this is fine, although not great style... usually I use the name of the env var as the value for True (i.e. LLEAP in this case)

Reply:
Ideally I would probably write the statement as
`lly = (os.getenv('LLEAP', '') == 'LLEAP' )`

Reply:
Also, right now, you will get an error if LLEAP is not defined; you can mitigate that by using os.getenv with an empty string as default, but then we would still have the possibility that lly does not get defined, if LLEAP is something other than '' or '--noleap', so adding something like `else: raise ValueError(os.environ['LLEAP']) `would be good.

Reply:
Ok. I fixed all of this.